### ISSUE TITLE
Save location files in nested folder structures as well

### ISSUE THREAD

**Issue by michaelbornholdt** [cytomining/DeepProfiler#315]
Related to:
https://github.com/cytomining/DeepProfiler/issues/264

I am not sure if this is an issue stemming from CellProfiler output? 

But why do we not sort the location files in the same way as all other data, ie by site? 

Currently my LINCS data and the example files have the old structure: 

![image](https://user-images.githubusercontent.com/56402523/142293633-9667d7a2-0543-4e72-8a1c-7edb55017d93.png)

**Comment by jccaicedo** [comment]
Yes, we should be consistent all across functionalities. This is the legacy structure.
It would be great to have a script that creates these files automatically for us, but that is a separate issue.

We should be able to make this change together with @Arkkienkeli
